pages/en/contrib/code-contrib.md

```diff
@@ -8,96 +8,5 @@ sidebar: contrib_sidebar
 permalink: /doc/en/contrib/code-contrib.html
 ---
 
-## How to contribute to the code
-
-You can contribute in a number of ways:
```

Comment:
Most (if not all) of the instructions in this file apply to all versions of LoopBack (2.x, 3.x, 4.x).

I am proposing to make `code-contrib-lb2-lb3.md` smaller and focus only on things specific to LB 3.x.

Thoughts?

Reply:
sounds good to me.